src/components/Filter.tsx

```diff
@@ -75,7 +75,7 @@ const FilterOption = styled.li`
   }
 `
 
-interface Props {
```

Comment:
It is unnecessary to change local type names. It makes the dev experience worse in many cases because it complicates copy and pasting between components